app/models/settings.rb

```diff
@@ -32,6 +32,7 @@ class Settings
   field :auto_associate_pocs, type: Boolean, default: false
   field :auto_confirm, type: Boolean, default: true
   field :bundle_file_path, type: String, default: 'temp/bundles'
+  field :vendor_file_path, type: String, default: 'temp/vendors'
```

Comment:
This is picky, but "vendors" aren't being stored here. It's vendor patients... worth a rename?

Reply:
done